dotnet-desktop-guide/net/winforms/overview/index.md

```diff
@@ -36,7 +36,7 @@ To learn how to create a Windows Forms app, see [Tutorial: Create a new WinForms
 
 ## Why migrate from .NET Framework
 
-Windows Forms for .NET 7.0 provides new features and enhancements over .NET Framework. For more information, see [What's new in Windows Forms for .NET 7](../whats-new/net70.md) and [.NET 5](../whats-new/net50.md). To learn how to migrate an app, see [How to migrate a Windows Forms desktop app to .NET 5](../migration/index.md).
+Windows Forms for .NET provides new features and enhancements over .NET Framework. For more information, see [What's new in Windows Forms for .NET 7](../whats-new/net70.md). To learn how to migrate an app, see [How to upgrade a Windows Forms desktop app to .NET 7](../migration/index.md)
```

Comment:
"migrate" is still in the h2

```suggestion
Windows Forms for .NET provides new features and enhancements over .NET Framework. For more information, see [What's new in Windows Forms for .NET 7](../whats-new/net70.md). To learn how to upgrade an app, see [How to upgrade a Windows Forms desktop app to .NET 7](../migration/index.md)
```

Reply:
I think that's OK since it's in the context of LEAVING .NET Framework while the other links are GOING to .NET